docs/demo.mdx

```diff
@@ -0,0 +1,506 @@
+import HomepageHeader from '@site/src/components/Homepage/Header/HomepageHeader';
+
+import '@site/src/css/custom.css';
+import '@site/node_modules/infima/dist/css/default/default.css';
+
+import FlavorizedCodeBlock from '@site/src/components/FlavorizedCodeBlock';
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# H1 heading
+
+## H2 heading
+
+### H3 heading
+
+#### H4 heading
+
+##### H5 heading
+
+###### H6 heading
+
+## Admonitions:
+
+:::caution CAUTION
+
+This is a CAUTION admonition with a custom background color, and a border color.
+
+:::
+
+:::danger DANGER
+
+This is a DANGER admonition with a custom background color, and a border color.
+
+:::
+
+:::info INFO
+
+This is an INFO admonition with a custom background color, and a border color.
+
+:::
+
+:::success TIP
+
+This is a TIP admonition with a custom background color, and a border color.
+
+:::
+
+:::note NOTE
+
+This is a NOTE admonition with a custom background color, and a border color.
+
+:::
+
+#### _BELOW is an admonition with code span and a link:_
+
+:::tip
+If you see a message like `command not found: detox`, make sure you have [installed Detox command line tools](#1-command-line-tools-detox-cli).
```

Comment:
I prefer not to use a real content from our docs, but something like [lorem-ipsum generated content](https://www.lipsum.com/). This will emphasize that it's not a real documentation page. I don't want someone to enter this from Algolia Docsearch or something when looking for our real docs.

Reply:
Throughout this page.

Reply:
For code blocks, we should have real syntax "dummy" functions (but something that is clearly just an example)

Reply:
Done! Thanks. I replaced all code syntax with "Dummy" syntax and all text with Lorem Ipsum :)